Meta/shell_include.sh

```diff
@@ -60,3 +60,26 @@ ensure_ladybird_source_dir() {
         export LADYBIRD_SOURCE_DIR
     fi
 }
+
+get_build_dir() {
+    ensure_ladybird_source_dir
+
+    # Note: Keep in sync with buildDir defaults in CMakePresets.json
+    case "$1" in
+        "default")
+            BUILD_DIR="${LADYBIRD_SOURCE_DIR}/Build/ladybird"
+            ;;
+        "Debug")
+            BUILD_DIR="${LADYBIRD_SOURCE_DIR}/Build/ladybird-debug"
+            ;;
+        "Sanitizer")
+            BUILD_DIR="${LADYBIRD_SOURCE_DIR}/Build/ladybird-sanitizers"
+            ;;
+        *)
+            echo "Unknown BUILD_PRESET: '$1'" >&2
+            exit 1
+            ;;
+    esac
+
+    echo "${BUILD_DIR}"
```

Comment:
Is this echo left over from testing the script?

Reply:
It's for the caller to do the `BUILD_DIR=` part: `BUILD_DIR=$(get_build_dir "$BUILD_PRESET")`

Reply:
Ah ok, my shell-noobishness is showing. :sweat_smile:

Reply:
I am no expert myself honestly!